packages/cli/src/constructs/browser-check.ts

```diff
@@ -113,7 +113,6 @@ export class BrowserCheck extends Check {
       script: this.script,
       scriptPath: this.scriptPath,
       dependencies: this.dependencies,
-      sourceFile: this.__checkFilePath ?? this.scriptPath,
```

Comment:
We used to use these for the `checkly test` command. Isn't that the case anymore?

Reply:
@umutuzgur , I started to refactor the Check constructs to handle internal attributes properly, but I think it will require more time to investigate all backward compatibility. Now, I simply loop the project's checks before deploying them and remove `__checkFilePath` and `sourceFile` interval attributes.